bindings/dart/rust/Cargo.toml

```diff
@@ -21,7 +21,7 @@ name = "opendal-dart"
 version = "0.1.0"
 
 [dependencies]
-flutter_rust_bridge = "2.8.0"
+flutter_rust_bridge = "=2.8.0"
```

Comment:
Do we need to regenerate the code?

Reply:
And do we need to pin the version at dart side?

Reply:
`flutter_rust_bridge_codegen generate` is used to generate the code, flutter_rust_bridge is the bridge.

https://github.com/apache/opendal/blob/main/bindings/dart/rust/src/frb_generated.rs#L29

the generated code need this.

Reply:
> And do we need to pin the version at dart side?

the version is be written into the code. So pin it is better. https://github.com/apache/opendal/blob/a7258381e8b12f64fe182e270cbc05c4f0d9ff9c/bindings/dart/rust/src/frb_generated.rs#L40

Reply:
Hi, the error in CI is:

```
thread '<unnamed>' panicked at src/frb_generated.rs:45:1:
assertion `left == right` failed: Please ensure flutter_rust_bridge's codegen (2.8.0) and runtime (2.9.0) versions are the same
  left: "2.8.0"
 right: "2.9.0"
```

I'm afraid that pin `2.8.0` doesn't work.

Reply:
Oh, I see. The issue here is rust side runtime version is too new..